packs/st2cd/actions/tag_branch.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+GIT=`which git`
+REPO=$1
+DATE=`date +%s`
+BRANCH=$2
+OUTPUT=/tmp/git-output-$DATE
+
+cd ${REPO}
+OUT=`$GIT checkout ${BRANCH} -q && $GIT pull origin ${BRANCH} -q > ${OUTPUT}`
+if [[ $? == 0 ]]
+then
+  NEWTAG=`git describe --tags --abbrev=0 | awk -F '[ .]' '{print $1"."$2"."$3+1}'`
+  if [[ $? == 0 ]]
+  then
+    OUT=`$GIT tag -a ${NEWTAG} -m "Creating tag ${NEWTAG} for branch ${BRANCH}" && $GIT push origin ${NEWTAG} -q`
```

Comment:
Also, I do think we should be explicit and specify a revision (I always do that when I do it manually) to make sure the correct revision is tagged.